boards/arc/nsim/nsim_em.dts

```diff
@@ -8,9 +8,24 @@
 
 #include "nsim_em.dtsi"
 
+#define XCCM_ADDR c0000000
+#define XCCM_SIZE DT_SIZE_K(64)
+#define YCCM_ADDR e0000000
+#define YCCM_SIZE DT_SIZE_K(64)
+
 / {
 	model = "snps,nsim_em";
 	compatible = "snps,nsim_em";
+
+	xccm0: xccm@XCCM_ADDR {
+		compatible = "arc,xccm";
+		reg = <DT_ADDR(XCCM_ADDR) XCCM_SIZE>;
+	};
+
+	yccm0: yccm@YCCM_ADDR {
+		compatible = "arc,yccm";
+		reg = <DT_ADDR(YCCM_ADDR) YCCM_SIZE>;
+	};
```

Comment:
Given that we don't actually use XY on `nsim_em` platform and we add `nsim_em11d` in DSP PR for DSP and XY testing - I'd propose to avoid unnecessary `nsim_em` platform changes.

Same for boards/arc/nsim/support/nsim_em.props & boards/arc/nsim/support/mdb_em.args files.